services/src/main/scala/cromwell/services/metadata/impl/MetadataServiceActor.scala

```diff
@@ -46,7 +46,10 @@ final case class MetadataServiceActor(serviceConfig: Config, globalConfig: Confi
 
   private val metadataSummaryRefreshLimit = serviceConfig.getOrElse("metadata-summary-refresh-limit", default = 5000)
 
-  val readActor = context.actorOf(ReadMetadataActor.props(), "read-metadata-actor")
+  private val metadataReadTimeout: Duration =
+    serviceConfig.getAs[FiniteDuration]("metadata-read-query-timeout").getOrElse(Duration.Inf)
```

Comment:
Any objection to letting folks explicitly list the configuration value to `metadata-read-query-timeout = Inf`? Would require switching this line to
`serviceConfig.getOrElse("metadata-read-query-timeout", Duration.Inf)`